src/app/components/PasswordForm/index.tsx

```diff
@@ -135,25 +123,16 @@ export default function PasswordForm<
           <TextField
             id="passwordConfirmation"
             label={t("confirm_password.label")}
-            type={passwordConfirmationView ? "text" : "password"}
+            type={passwordConfirmationViewVisible ? "text" : "password"}
             required
             onChange={handleChange}
             onBlur={validate}
             endAdornment={
-              <button
-                type="button"
-                className="flex justify-center items-center w-10 h-8"
-                tabIndex={-1}
-                onClick={() =>
-                  setPasswordConfirmationView(!passwordConfirmationView)
-                }
-              >
-                {passwordConfirmationView ? (
-                  <HiddenIcon className="h-6 w-6" />
-                ) : (
-                  <VisibleIcon className="h-6 w-6" />
-                )}
-              </button>
+              <PasswordViewAdornment
```

Comment:
Just on the naming here normally we start with a Verb "ViewPasswordAdornment", but I wonder if "TogglePasswordVisibilityAdornment" or something would be more accurate (I know it is a bit long)

Reply:
didn't we follow a similar convention for qrcode scanner component? name was ```QrcodeAdornment``` what we want to specify action in the name there?

viewpassword - action.
passwordView -  adornment for the password view component. makes sense?